src/cursor.c

```diff
@@ -292,13 +291,15 @@ PyObject *cursor_fetchone(CursorObject *cursor, PyObject *args) {
     int has_more_second = 0;
     int fetch_status_first =
         connection_fetch(cursor->conn, &row, &has_more_first);
-    int fetch_status_second =
-        connection_fetch(cursor->conn, NULL, &has_more_second);
+    int fetch_status_second = 0;
+    if (fetch_status_first == 1) {
+      fetch_status_second =
+          connection_fetch(cursor->conn, NULL, &has_more_second);
+    }
```

Comment:
This is a small fix for fetches in lazy mode without any result. See the new tests and the [failed actions run](https://github.com/memgraph/pymgclient/runs/2351308621?check_suite_focus=true).

If the first fetch receives the `SUCCESS` message, then the second fetch failed before. It didn't occurred when the cursor could fetch at least one row, because then the status of the cursor was set to `CURSOR_STATUS_READY` when the last row is fetched, and the check around line 268 handled this case.